test/integration/notification_settings_test.rb

```diff
@@ -46,7 +46,7 @@ class NoticationSettings < SystemTest
 
     visit edit_profile_path(as: user)
 
-    assert_no_text I18n.t("profiles.edit.notifier.email_notifications")
+    assert_no_text I18n.t("notifiers.show.title")
```

Comment:
Looks good.

As an aside, I just noticed that `NoticationSettings` is an old typo, missing an `f` and a `Test`-suffix. Would you be up for fixing it?

Reply:
Sure!

Reply:
Thanks, I think it was my typo originally 🤦‍♂ , much appreciated 👍!